cmd/derper/derper.go

```diff
@@ -190,29 +196,33 @@ func main() {
 	s.SetTCPWriteTimeout(*tcpWriteTimeout)
 
 	if *dev {
-		k := setec.StaticSecret(os.Getenv("TAILSCALE_DERPER_MESH_KEY"))
-		if k.GetString() != "" {
-			key, err := checkMeshKey(k.GetString())
-			if err != nil {
-				log.Fatalf("setec static failed to get key: %v", err)
-			}
-			s.SetMeshKey(key)
-		} else {
+		key, ok := os.LookupEnv("TAILSCALE_DERPER_MESH_KEY")
+		if !ok || key == "" {
 			log.Println("No mesh key specified for dev")
+		} else if meshKey, err := checkMeshKey(key); err != nil {
+			log.Fatalf("Invalid mesh key from environment: %v", err)
+		} else {
+			s.SetMeshKey(meshKey)
+			log.Println("Set mesh key from TAILSCALE_DERPER_MESH_KEY")
 		}
 	} else if *secretsURL != "" {
+		meshKeySecret := strings.Join([]string{strings.Trim(*secretPrefix, "/"), "meshkey"}, "/")
```

Comment:
We could use `path.Join(*secretPrefix, setecMeshKeyName)` [link](godoc.org/path#Join) to simplify this.